Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "SQLite.swift",
+        "repositoryURL": "https://github.com/stephencelis/SQLite.swift.git",
```

Comment:
Is this auto generated, do we need if we are removing SQLite.swift from the lib?

Reply:
Ahh I will remove that, it was automatically generated.